express/scripts/html-sanitizer.js

```diff
@@ -0,0 +1,182 @@
+const HtmlSanitizer = new (function handSanitizer() {
```

Comment:
this whole file is only used by the content-replace. So why is it in a separate file? Let's move it into content-replace.js and save on requests.